src/common/configs/needs-review-rule-resources.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const needsReviewRuleResourcesPath =
+    'https://accessibilityinsights.io/info-examples/web/needs-review';
+
+export const isOutcomeNeedsReview = (ruleId: string, outcomeType: string) => {
+    return needsReviewRules.includes(ruleId) && outcomeType === 'review' ? true : false;
+};
```

Comment:
Does `needsReviewRules` even need to exist? It seems like this could potentially be simplified to only consider the outcomeType:

```suggestion
export const isOutcomeNeedsReview = (outcomeType: string): boolean => {
    return outcomeType === 'review';
};
```

Reply:
Some of the rules which sometime come to needs review tab don't have their respective documentation URL in the need review URL so `needsReviewRules` is required to check if page exist or not otherwise it will give 404 error. 
![image](https://github.com/microsoft/accessibility-insights-web/assets/150002431/a9738833-c830-453b-8b1b-eb7015992c98)
![image](https://github.com/microsoft/accessibility-insights-web/assets/150002431/2d55c12b-ffb8-4c20-b6b6-3767c4962d77)

Reply:
@v-sharmachir, let me dig into the label-content-name-mismatch issue. If it appears only in Needs Review, then I'd argue that its documentation should also appear there. Ultimately, I'm hoping to avoid having 2 separate places that need updating if a rule moves between automated checks and needs review.

Reply:
removed the `needsReviewRules` and updated the test files as well